pkg/image/importer/importer.go

```diff
@@ -282,6 +282,27 @@ func applyErrorToRepository(repository *importRepository, err error) {
 	}
 }
 
+func calculateImageSize(ctx gocontext.Context, repo distribution.Repository, image *api.Image) error {
+	bs := repo.Blobs(ctx)
+
+	layerSet := sets.NewString()
+	size := int64(0)
+	for i := range image.DockerImageLayers {
+		layer := &image.DockerImageLayers[i]
+		desc, err := bs.Stat(ctx, digest.Digest(layer.Name))
```

Comment:
Opened test change in https://github.com/openshift/origin/pull/9118

On Wed, Jun 1, 2016 at 3:41 PM, Alexey Gladkov <notifications@github.com>
wrote:

> In pkg/image/importer/importer.go
> <https://github.com/openshift/origin/pull/9115#discussion_r65428383>:
>
> > @@ -282,6 +282,27 @@ func applyErrorToRepository(repository *importRepository, err error) {
> >  	}
> >  }
> >
> > +func calculateImageSize(ctx gocontext.Context, repo distribution.Repository, image *api.Image) error {
> > +	bs := repo.Blobs(ctx)
> > +
> > +	layerSet := sets.NewString()
> > +	size := int64(0)
> > +	for i := range image.DockerImageLayers {
> > +		layer := &image.DockerImageLayers[i]
> > +		desc, err := bs.Stat(ctx, digest.Digest(layer.Name))
>
> I'll open a PR to disable the tests now and we can lower the priority.
>
> OK. Then I close this PR.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9115/files/492baa03f23cf45ca229c3501c29ca4bccf546f3#r65428383>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p_Cre1dOt6e_KNPE4GhqBex9VKe5ks5qHeB4gaJpZM4IrzC2>
> .
>